SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/Storage.cs

```diff
@@ -276,12 +260,15 @@ private void ProcessLogIds(string channelName, string batchId, IEnumerable<Tuple
             AppCenterLog.Debug(AppCenterLog.LogTag, message);
         }
 
-        private async Task InitializeDatabaseAsync()
+        private void InitializeDatabase()
         {
+            Directory.CreateDirectory(Constants.AppCenterFilesDirectoryPath);
```

Comment:
`Directory.CreateDirectory()` can throw IOException. Why not move it inside the try block?